base_contact/README.rst

```diff
@@ -0,0 +1,45 @@
+Contacts Management
+===================
+
+This module was written to extend the contact management functionality. It
+allows you to:
+
+    - set several job positions in different companies per contact,
+    - set some extra informations like birthdate and nationality.
+
+Usage
+=====
+
+New tabs 'Other Positions' and 'Personal Information' are available on the
+contact form.
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
+
+ * Migrate to the new API
+
+Credits
+=======
+
+Contributors
+------------
+
+* OpenERP SA
+* Sebastien Alix <sebastien.alix@osiell.com>
```

Comment:
Yes, please, add the other contributors as this is the first time they are summarized. Open source will reward you :wink: